waiter/src/waiter/scheduler.clj

```diff
@@ -362,12 +362,12 @@
   "Queries the scheduler and builds a list of healthy and unhealthy instances for the specified service-id."
   [service-id service-instances]
   (when service-instances
-    (let [healthy-instances (vec (filter :healthy? service-instances))
-          unhealthy-instances (vec (clojure.set/difference (set service-instances) (set healthy-instances)))]
+    (let [{healthy-instances true unhealthy-instances false}
+          (group-by (comp boolean :healthy?) service-instances)]
```

Comment:
Nitpick: Can we move this to the previous line and have a comma as follows: `{healthy-instances true, unhealthy-instances false}`?

Reply:
I'm OK with that. Do we have any guidelines on what to do about really long lines? I felt like this line was a bit too long, which is why I broke it onto two lines, but I don't really like doing that between let value–binding pairs.

Reply:
We don't have a strict rule with long lines, but we just use our judgment to split lines. It didn't feel to me that this line would be too long.